universe/Ship.cpp

```diff
@@ -431,6 +431,37 @@ float Ship::InitialPartMeterValue(MeterType type, const std::string& part_name)
     return 0.0f;
 }
 
+float Ship::CurrentPartClassMeterValue(MeterType type, ShipPartClass part_class) const {
```

Comment:
any suggestions for a name shorter than `SumCurrentPartMeterValuesForPartClass` highly welcome